frontend/packages/gitops-plugin/src/components/list/GitOpsListItem.tsx

```diff
@@ -2,32 +2,42 @@ import * as React from 'react';
 import * as _ from 'lodash';
 import { useTranslation } from 'react-i18next';
 import { Grid, GridItem, Card, CardBody } from '@patternfly/react-core';
-import { history, ResourceLink } from '@console/internal/components/utils';
+import { history, ResourceLink, ExternalLink } from '@console/internal/components/utils';
+import { K8sResourceKind } from '@console/internal/module/k8s';
 import { GitOpsAppGroupData } from '../utils/gitops-types';
 import './GitOpsListItem.scss';
 
 interface GitOpsListItemProps {
   appGroup: GitOpsAppGroupData;
+  argocdLink?: K8sResourceKind;
 }
 
-const GitOpsListItem: React.FC<GitOpsListItemProps> = ({ appGroup }) => {
+const GitOpsListItem: React.FC<GitOpsListItemProps> = ({ appGroup, argocdLink }) => {
   const { t } = useTranslation();
   const handleCardClick = () => {
     history.push(`/environments/${appGroup.name}?url=${appGroup.repo_url}`);
   };
 
   return (
-    <Card className="odc-gitops-list-item" onClick={handleCardClick} isHoverable>
+    <Card className="odc-gitops-list-item" isHoverable>
       <CardBody>
         <Grid className="odc-gitops-list-item__body">
-          <GridItem lg={6} md={6} sm={6}>
+          <GridItem lg={5} md={5} sm={5} onClick={handleCardClick}>
             <ResourceLink kind="application" name={appGroup.name} linkTo={false} />
           </GridItem>
-          <GridItem lg={6} md={6} sm={6}>
+          <GridItem lg={5} md={5} sm={5} onClick={handleCardClick}>
             {t('gitops-plugin~{{count, number}} Environment', {
               count: _.size(appGroup.environments),
             })}
           </GridItem>
+          <GridItem lg={2} md={2} sm={2}>
+            {argocdLink && (
+              <ExternalLink
+                href={`${argocdLink.spec.href}/applications?labels=app.kubernetes.io%252Fname%253D${appGroup.name}`}
```

Comment:
@keithchong please create a utility function in `gitops-utils.ts` for generating this url as desired. You can see `getPipelinesBaseURI` for reference.

Reply:
Updated.